### PR TITLE
Update metadata.json

### PR DESCRIPTION
**Pull request by Zerlingg**

### CODE REVIEW

**Review by ririxidev — APPROVED** [review]

File: websites/A/AnimeFLV/dist/metadata.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   },
   "url": [
     "www3.animeflv.net",
-    "animeflv.ru"
+    "animeflv.ac"
   ],
   "version": "1.1.8",
   "logo": "https://i.ibb.co/k1x0mXZ/descarga.png",
```
Comment (ririxidev) [inline websites/A/AnimeFLV/dist/metadata.json:22]: could you upload logo and thumbnail to imgur thanks heh

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/A/AnimeFLV/dist/metadata.json (hunk 1 above)
Comment (Zerlingg) [reply websites/A/AnimeFLV/dist/metadata.json]: Done!

**Review by Zerlingg — COMMENTED** [review]
Added new lenguage :-)

File: websites/A/AnimeFLV/dist/metadata.json [hunk 2]
```diff
@@ -12,15 +12,17 @@
...
   "url": [
     "www3.animeflv.net",
+    "animeflv.ac",
     "animeflv.ru"
   ],
-  "version": "1.1.8",
-  "logo": "https://i.ibb.co/k1x0mXZ/descarga.png",
-  "thumbnail": "https://i.ibb.co/HFKzffL/05212f589cb42f3d29073ce164a66eaa.jpg",
+  "version": "1.1.9",
+  "logo": "https://i.imgur.com/QmFuSUM.png",
```
Comment (Bas950) [inline websites/A/AnimeFLV/dist/metadata.json:24]: please get a logo thats 512x512 or bigger

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/A/AnimeFLV/dist/metadata.json [hunk 3]
```diff
@@ -16,11 +16,12 @@
...
   "url": [
     "www3.animeflv.net",
-    "animeflv.ac"
+    "animeflv.ac",
+    "animeflv.ru"
   ],
   "version": "1.1.8",
-  "logo": "https://i.ibb.co/k1x0mXZ/descarga.png",
-  "thumbnail": "https://i.ibb.co/HFKzffL/05212f589cb42f3d29073ce164a66eaa.jpg",
+  "logo": "https://i.imgur.com/QmFuSUM.png",
```
Comment (ririxidev) [inline websites/A/AnimeFLV/dist/metadata.json]: 512x512

**Review by ririxidev — CHANGES_REQUESTED** [review]
change this and we are done heh

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**